client/js/lounge.js

```diff
@@ -1039,9 +1042,7 @@ $(function() {
 			socket.emit("names", {target: self.data("id")});
 		}
 
-		if (screen.width > 768 && chan.hasClass("chan")) {
-			input.focus();
-		}
+		focus();
```

Comment:
@YaManicKill what you think? Should we keep (re-open in reality) the keyboard after clicking the send button?